crm-app/src/component/containers.js

```diff
@@ -0,0 +1,36 @@
+import { connect } from 'react-redux';
+import { addCustomer, updateCustomer, deleteCustomer, displayCustomer, resetCustomer } from '../actions';
+import CustomerList from './ui/CustomerList';
+import ProfileView from './ui/ProfileView';
+
+export const List = connect(
+	state => ({
+		customers: state.customers
+	}),
+	dispatch => ({
+		onDisplayItem(index) {
+			dispatch(displayCustomer(index))
+		}
+	})
+)(CustomerList);
+
+export const View = connect (
+	state => ({
+		displayedIndex: state.displayedIndex,
+		customers: state.customers
+	}),
+	dispatch => ({
+		onAdd(name, mail, phone) {
+			dispatch(addCustomer(name, mail, phone))
+		},
+		onUpdate(index, name, phone) {
+			dispatch(updateCustomer(index, name, phone))
+		},
+		onCancel() {
+			dispatch(resetCustomer())
```

Comment:
It would be better to use only one of `cancel` and `reset` words.

Reply:
Done